src/GitHub/Endpoints/Activity/Starring.hs

```diff
@@ -55,3 +58,13 @@ myStarred auth =
 -- | All the repos starred by the authenticated user.
 myStarredR :: Maybe Count -> Request 'True (Vector Repo)
 myStarredR = PagedQuery ["user", "starred"] []
+
+
+-- | All the repos starred by the authenticated user.
+myStarredAcceptStar :: Auth -> IO (Either Error (Vector RepoStarred))
+myStarredAcceptStar auth =
+    executeRequest auth $ myStarredAcceptStarR Nothing
+
+-- | All the repos starred by the authenticated user.
```

Comment:
Please add `see` link. Like in `reposStarredByR`. 

Could you also add to `myStarredR` too, looks like it's missing as well.